.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -27,6 +27,7 @@ body:
         - Workers Runtime
         - Wrangler core
         - Miniflare
+        - KV Asset Handler
```

Comment:
Super nitpicky, but we should standardise on title case or sentence case (looking at Wrangler core above)

Reply:
@penalosa yeah i thought about that -- my thinking was that kv asset handler is the actual proper name of a package so it should be all capitalized, but that wrangler core isn't really an officially recognized entity on its own -- maybe we just change it to say `Wrangler` or `Wrangler (core)`?